samples/tests/ComponentExamples/ComponentExamples.test.ts

```diff
@@ -58,6 +58,8 @@ test.describe('Component Examples Test', () => {
       return document.querySelectorAll('video').length === 0;
     });
 
+    await page.mouse.move(0, 0);
```

Comment:
Please add a comment above this saying why we need to move the mouse away. Something like, "Move the mouse off-screen to avoid tooltips / delayed focus from introducing flake in the snapshots).

Reply:
Added comment on newest commit:
// Move the mouse off-screen to avoid tooltips / delayed focus from introducing flake in the snapshots